systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaRebalanceUtils.java

```diff
@@ -6,8 +6,11 @@
 
 import io.strimzi.api.kafka.model.KafkaRebalance;
 import io.strimzi.api.kafka.model.status.Condition;
+import io.strimzi.api.kafka.operator.assembly.KafkaRebalanceAnnotation;
 import io.strimzi.api.kafka.operator.assembly.KafkaRebalanceState;
+import io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceAssemblyOperator;
```

Comment:
What is this import used for? I think it would be much better to keep clean separation between the system tests and internal code of the cluster operator.

Reply:
Yeah I was thinking about that too. But this was imported after https://github.com/strimzi/strimzi-kafka-operator/pull/3397#pullrequestreview-456432182

Reply:
So maybe I should create some constant?

Reply:
I think this is similar to the enums for example. We expect the users to use the annotation but we want them to hardcode it in code. If that is good enough for the users, it is for sure good enough for the system tests. I think it should be either moved to the `api` module somewhere or it should be just hardcoded here instead of depending on the `cluster-operator` module.